diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index 26c2a65..96897fb 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -10,6 +10,7 @@
 
 const fs = require('fs');
 const path = require('path');
+const WebWorkerPlugin = require('worker-plugin');
 const webpack = require('webpack');
 const resolve = require('resolve');
 const PnpWebpackPlugin = require('pnp-webpack-plugin');
@@ -598,6 +599,7 @@ module.exports = function (webpackEnv) {
       ],
     },
     plugins: [
+      new WebWorkerPlugin(),
       // Generates an `index.html` file with the <script> injected.
       new HtmlWebpackPlugin(
         Object.assign(
